minecraft_servers/fantasy/manifest.json

```diff
@@ -0,0 +1,35 @@
+{
+  "server_name": "fantasy",
+  "nice_name": "Rede Fantasy",
+  "direct_ip": "redefantasy.com",
+  "server_wildcards": [
+    "%.redefantasy.com"
+  ],
+  "gamemodes": {
+    "prison": {
+      "name": "Prison Phoenix",
+      "color": "#FFAA00",
+      "command": "/serverconnector prison-phoenix"
+    }
+  },
+  "social": {
+    "discord": "https://discord.redefantasy.com",
```

Comment:
Discord has to be https://discord.gg/[INVITE]

Reply:
Resolved